build.gradle

```diff
@@ -1,36 +1,27 @@
 plugins {
-	id 'fabric-loom' version '0.12.+'
-	id 'io.github.juuxel.loom-quiltflower' version '1.7.+'
-	id 'org.quiltmc.quilt-mappings-on-loom' version '4.0.0'
+	id 'fabric-loom' version '1.+'
```

Comment:
This should probably be `1.0.+`, since 1.1.x is a different development series.

Reply:
That was my impression too, but I was mimicking [what Aurora did in SpruceUI](https://github.com/LambdAurora/SpruceUI/blob/1.19.3/build.gradle#L2). I can totally change it tho!